db/install.xml

```diff
@@ -15,10 +15,10 @@
         
         <!-- Instructions Fields -->
         <FIELD NAME="observer_ins" TYPE="text" SEQUENCE="false" NOTNULL="false" COMMENT="string encoded observer instructions"/>
-        <FIELD NAME="observer_ins_f" TYPE="char" LENGTH="32" SEQUENCE="false" NOTNULL="false" COMMENT="Format for the observer instructions, either 'html', 'markdown' or 'plaintext'"/>
+        <FIELD NAME="observer_ins_f" TYPE="int" LENGTH="20" SEQUENCE="false" NOTNULL="false" COMMENT="Format for the observer instructions, 1 = 'html'"/>
```

Comment:
Do these actually need to be length 20? I would restrict this, just to make the DB operations a tiny bit better